### PR TITLE
Admin Center - Quotas are found under Storage not Files

### PR DESCRIPTION
**Pull request by deyjcode**
To edit or update a quota, one needs to open the Storage node - not the Files node.

In addition, it is misleading to suggest that all FSRM options are available. Setting quota management, denying disk space to users, and limiting disk usage is all that is available. 

Modifying FSRM e-mail templates, SMTP settings, or any other additional FSRM feature has not been added to Admin Center yet.

### CODE REVIEW

**Comment by daveba** [comment]
@JasonGerend, please review and if you agree with the changes, please merge.

File: WindowsServerDocs/manage/windows-admin-center/use/manage-servers.md [hunk 1]
```diff
@@ -429,6 +428,7 @@ The following features are supported in Storage:
 - Edit volume properties
 - Delete a volume
 - Install Quota Management
```
Comment (JasonGerend) [inline WindowsServerDocs/manage/windows-admin-center/use/manage-servers.md]:
```suggestion
```

File: WindowsServerDocs/manage/windows-admin-center/use/manage-servers.md [hunk 2]
```diff
@@ -429,6 +428,7 @@ The following features are supported in Storage:
...
+- Manage File Server Resource Manager Quotas [Storage->Create/Update Quota](https://docs.microsoft.com/windows-server/storage/fsrm/quota-management)
```
Comment (JasonGerend) [inline WindowsServerDocs/manage/windows-admin-center/use/manage-servers.md]:
```suggestion
- [Quota Management](../../storage/fsrm/quota-management.md)
```

**Comment by daveba** [comment]
@JasonGerend Are we good to merge this pull request?

**Comment by JasonGerend** [comment]
@daveba Yes, let's merge it - I was waiting to make sure that the original contributor was OK with my tweaks, but if the original contributor has objections, they can open a new PR for further refinements. Thanks!

**Comment by deyjcode** [comment]
> 
> 
> @daveba Yes, let's merge it - I was waiting to make sure that the original contributor was OK with my tweaks, but if the original contributor has objections, they can open a new PR for further refinements. Thanks!

Apologies. I was fine with it and forgot to reply.

**Pull request merged**